### DISCUSSION TITLE
Changing color of active menu item without affecting the color of disabled things

### DISCUSSION THREAD

**Discussion by pcherna** [saadeghi/daisyui#3183] [Q&A] [answered]
Using the default light theme, the active color for menu items is quite stark. It's easy to style that by replacing the neutral color in the theme, however that ends up recoloring disabled buttons, which I don't want. See example:

https://play.tailwindcss.com/HyLyCjzpbg

I also tried leaving the theme alone but using explicit color classes on the item, but I couldn't figure out how to affect the color while you're pressing down on a non-active item. Even if I could, there are a ton of classes I would need to apply for lots of different states, whereas using neutral just works.

Is there a better way?

**Comment by condemil** [comment, marked as answer]
One of the options is to use another color for active menu item, for example secondary color. You can search for `active` in sources [here](https://github.com/saadeghi/daisyui/blob/master/src/components/styled/menu.css), copy-paste required part to your css file and replace color there.

**Reply by pcherna** [reply]
OK thank you. In a quick test I see that does give me control. I do think this is worth making more directly controllable, but I'm unblocked now, thank you.
